docs/faq.md

```diff
@@ -57,18 +57,6 @@ bundle install --no-deployment --without development --path .bundle
 bundle exec librarian-puppet install --path=shared --clean
 ```
 
-### Q: What's a good approach to merging our-boxen back into my private fork?
```

Comment:
I'm wondering whether this is still helpful for those who aren't upgrading but trying to get back onto the golden path?